modules/http/http.c

```diff
@@ -390,6 +390,7 @@ _format_request_headers(HTTPDestinationDriver *self, LogMessage *msg)
   struct curl_slist *headers = NULL;
   GList *l;
 
+  headers = _add_header(headers, "Expect", "");
```

Comment:
How does this work? Does Libcurl detect internally that expect header was already set? Or two expect headers will be sent in the header, and this one overrides the other one?